docs/src/main/sphinx/functions/list-by-topic.rst

```diff
@@ -0,0 +1,548 @@
+=========================================
+List of functions by topic
+=========================================
+
+Aggregate
+---------
+
+Details in :doc:`aggregate`
```

Comment:
Should be an action item:

For more details, see :doc:

Reply:
I agree.